### ISSUE TITLE
Path Descriptions

### ISSUE THREAD

**Issue by amstaker** [AtomLinter/linter-phpcs#189]
Can you include in the documentation the proper way to setup and configure this for a windows user.  There is way too much ambiguity which causes a bunch of headaches.

In particular the Executable Path and Code Standard or Config File fields are unclear.

- Because you cannot simply select a file what does the path look like? For example:  
--C:\Users\%moeszyslak%\AppData\Roaming\Composer\vendor\squizlabs\php_codesniffer\phpcs.bat OR
--C:\Users\%moeszyslak%\AppData\Roaming\Composer\vendor\bin OR
--C:\Users\%moeszyslak%\AppData\Roaming\Composer\vendor\bin\phpcs
--OR something entirely different?

It's not clear whether you need to reference a path and/or a specific file.  The same applies for the code standard or config file fields.  

Using `composer global show -P` I get the following output:

C:\Users\%moeszyslak%\AppData\Roaming\Composer\vendor\drupal\coder
C:\Users\%moeszyslak%\AppData\Roaming\Composer\vendor\squizlabs\php_codesniffer
(%moeszyslak% is a fake user name).

I can execute `phpcs -v` as well.

Any documentation updates, field updates and/or clarity here would be appreciated.

**Comment by Arcanemagus** [comment]
# Executable Path

Executable path should be set to just `phpcs` (the default) if it is on your `%PATH%`, this will let the system resolve it for you. If that isn't working then it should be set to the _full_ path to `phpcs`, i.e. the output of `where.exe phpcs`. For example on my system it would be set to:

```text
C:\Users\Landon Abney\AppData\Roaming\Composer\vendor\bin\phpcs.bat
```

# Code Standard or Config File

This is simply passed onto `phpcs` directly via the `--standard` CLI option, see the relevant [`phpcs` documentation](https://github.com/squizlabs/PHP_CodeSniffer/wiki/Usage#specifying-a-coding-standard) for how that works.

Note that if you enable "Search for configuration files" and this package finds any of `phpcs.xml`, `phpcs.xml.dist`, `phpcs.ruleset.xml` or `ruleset.xml` it will use that and ignore anything you have put in the "Code Standard or Config File" field.

----

Hopefully that clarifies things for you, leaving this open as the settings could be clearer.

**Comment by amstaker** [comment]
I've verified that my environmental variables are correct.  I have references to:

C:\Users\moesyzlak\AppData\Roaming\Composer\vendor\bin;
C:\bin

I also validated that using either the default phpcs or C:\Users\moesyzlak\AppData\Roaming\Composer\vendor\bin\phpcs.bat

I receive an error parsing PHPCS response:  Something went wrong attempting to parse the PHPCS output.  I don't get any more detail, is there any additional steps I can use to troubleshoot?
